web/src/components/CityContentLayout.tsx

```diff
@@ -56,12 +54,12 @@ const CityContentLayout = (props: CityContentLayoutProps): ReactElement => {
         />
       }
       footer={
-        !isLoading && showFooter && !viewportSmall ? (
-          <CityContentFooter city={city.code} language={languageCode} />
-        ) : null
+        viewportSmall
+          ? !isLoading && Toolbar
+          : showFooter && !isLoading && <CityContentFooter city={city.code} language={languageCode} />
```

Comment:
These changes render the toolbar also as footer on mobile which was necessary to fix the position of the player/chat button over the footer... took me ages to figure out a nice solution...